### PR TITLE
HAWQ-1182. Add Macro for unused argument and variable.

### PR DESCRIPTION
**Pull request by paul-guo-**

### CODE REVIEW

File: src/include/postgres.h [hunk 1]
```diff
@@ -513,6 +513,18 @@ extern void gp_set_thread_sigmasks(void);
...
+#ifndef POSSIBLE_UNUSED_VAR
+#define POSSIBLE_UNUSED_VAR(x)	((void)x)
+#endif
+
+#ifndef POSSIBLE_UNUSED_ARG
+#define POSSIBLE_UNUSED_ARG(x)	((void)x)
+#endif
+
+#ifndef UNUSED_ARG
+#define UNUSED_ARG(x)			((void)x)
```
Comment (xunzhang) [inline src/include/postgres.h]: ......indent

Comment (xunzhang) [inline src/include/postgres.h]: +1 except for the indent above

**Comment by wengyanqing** [comment]
LGTM

File: src/include/postgres.h (hunk 1 above)
Comment (paul-guo-) [inline src/include/postgres.h]: It is because the tab stop shown on this page is 8, while that in hawq 4.

**Pull request closed without merging**